Source/Core/DiscIO/VolumeWiiCrypted.cpp

```diff
@@ -67,6 +63,7 @@ bool CVolumeWiiCrypted::Read(u64 _ReadOffset, u64 _Length, u8* _pBuffer, bool de
 
 	FileMon::FindFilename(_ReadOffset);
 
+	std::vector<u8> read_buffer(s_block_total_size);
```

Comment:
I'd prefer having this as a member variable like before to avoid allocating this over and over.